bin/compile

```diff
@@ -42,8 +42,10 @@ fi
 echo "-----> Exporting sprettur configuration"
 mkdir -p $BUILD_DIR/.profile.d
 
-echo "export STACK=$STACK" >> $BUILD_DIR/.profile.d/sprettur.sh
-echo 'export PATH=$PATH:$HOME/.sprettur/bin/' >> $BUILD_DIR/.profile.d/sprettur.sh
+PROFILE_D=$BUILD_DIR/.profile.d/sprettur.sh
+echo "export STACK=$STACK" >> $PROFILE_D
+echo "export PATH=\$PATH:\$HOME/.sprettur/bin/" >> $PROFILE_D
+echo "export TERM=\${TERM:-dumb}" >> $PROFILE_D
```

Comment:
I've changed to the default to `dumb` to reflect the value that Bash automatically sets if it's not present. The main intent of this PR is to silence warnings that show up sometimes. Without understanding the implications of setting this to `xterm` and proper Dashboard support I think we should punt on that.